crates/editor/src/items.rs

```diff
@@ -1199,83 +1195,6 @@ pub fn active_match_index(
     }
 }
 
-pub struct CursorPosition {
```

Comment:
Move this file into `go_to_line/src/cursor_postion.rs` because is depended on `go_to_line` crate, but editor also depended on `editor` crate.